README.md

```diff
@@ -17,6 +17,7 @@
 
 ### The Solution
 
+- Segregated logical environments that can span across different kubernetes clusters
```

Comment:
I'm not a fan of `Segregated` here; it's got some connotations, some of which aren't all that positive.
```suggestion
- Separated logical environments that can span across different kubernetes clusters
```